src/Core_Command.php

```diff
@@ -246,16 +269,16 @@ function () use ( $temp ) {
 
 			if ( 'nightly' !== $version ) {
 				$md5_response = Utils\http_request( 'GET', $download_url . '.md5' );
-				if ( 20 !== (int) substr( $md5_response->status_code, 0, 2 ) ) {
-					WP_CLI::error( "Couldn't access md5 hash for release (HTTP code {$response->status_code})." );
-				}
-
-				$md5_file = md5_file( $temp );
+				if ( 200 === $md5_response->status_code ) {
```

Comment:
What about HTTP status code 203 ?
```
203 Non-Authoritative Information (since HTTP/1.1)
The server is a transforming proxy (e.g. a Web accelerator) that received a 200 OK
from its origin, but is returning a modified version of the origin's response.
```

Reply:
The intent here was fine, but I thought the execution was unnecessarily convoluted. Changed to check for any 2XX status codes in 2ad8092b.